src/test/regress/expected/multi_mx_call.out

```diff
@@ -0,0 +1,53 @@
+-- Test passing off CALL to mx workers
```

Comment:
These tests failed probably with a worker crash, when postgres/citus is compiled with `--enable-cassert`. Can you double check that?

Reply:
for me too, https://github.com/citusdata/citus/pull/2998/ fixes it